css/style.css

```diff
@@ -1,19 +1,352 @@
+  width: var(--page-width);
+  max-width: 100vw;
+  margin: 0 auto;
+}
+
+h1,
+h2,
+h3 {
+  font-weight: lighter;
+}
+
+h1 {
+  font-size: var(--fs-h1);
+  line-height: calc(var(--fs-h1) * 1.5);
+}
+
+h2 {
+  font-size: var(--fs-h2);
+  line-height: calc(var(--fs-h2) * 1.5);
+}
+
+h3 {
+  font-size: var(--fs-h3);
+  line-height: calc(var(--fs-h3) * 1.5);
+  font-weight: normal;
+}
+
+p {
+  font-size: var(--fs-p);
+  line-height: calc(var(--fs-p) * 1.5);
+}
+
+a {
+  color: var(--color-mid);
+  text-decoration: none;
+  transition: all 0.2s;
+}
+
+a:hover,
+a:focus {
+  color: var(--color-brand);
+  transition: all 0.2s;
+}
+
+.button {
+  margin-top: var(--gap-vertical);
+  padding: 1.25rem 2.75rem;
+  border-radius: 0.25rem;
+
+  background-color: var(--color-brand);
+  color: var(--color-light);
+  font-size: var(--fs-navbar);
+
+  transition: all 0.2s;
+}
+
+.button:hover {
+  background-color: var(--color-brand-dark);
+  color: var(--color-light);
+  transform: scale(1.05);
+  transition: all 0.2s;
+}
+
+/* Header */
+
+header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.25rem 8%;
+}
+
+header nav {
+  width: 50%;
+  min-width: 500px;
+  font-size: var(--fs-navbar);
+}
+
+header img {
+  height: 3rem;
+}
+
+#navbar__items {
+  display: flex;
+  justify-content: space-between;
+
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  color: var(--color-mid);
+}
+
+.navbar__item:first-child a:not(:hover):not(:focus) {
+  color: var(--color-dark);
+}
+
+.navbar__item:first-child a {
+  font-weight: bold;
+}
+
+/* Hero */
+
+#hero {
+  background: url("../img/first-background.jpg") no-repeat top center;
+  background-size: cover;
+  height: 96vh;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 2rem;
+
+  color: var(--color-light);
+
+  text-align: center;
+}
+
+#hero p {
+  font-size: var(--fs-hero-description);
+  line-height: var(--fs-hero-description);
+}
+
+/* Benefits */
+
+#benefits {
+  padding: 10rem 10% 8rem;
+  text-align: center;
+  width: 100%;
+}
+
+#benefits h2 {
+  margin-bottom: var(--gap-vertical);
+}
+
+#benefits-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  max-width: 100vw;
+}
+
+.benefit {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 280px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.benefit img {
+  height: 200px;
+}
 
+#to-store {
+  display: flex;
+  max-width: 100vw;
+  margin-bottom: 6rem;
+}
 
+#to-store img {
+  width: 40%;
+  min-width: none;
```

Comment:
AFAIK, `none` is not a valid value for `min-width`.

Reply:
You are rigth. The correct would be unset.
Thanks.